mockvulnerableapis/views.py

```diff
@@ -62,34 +94,13 @@ def insert_data(request):
 
     for obj in testData:
         try:
-            resp = obj.testData.responsePayload
-            headers = obj.testData.responseHeaders
-            try:
-                resp = vars(obj.testData.responsePayload)
-                for key in resp:
-                    try:
-                        resp[key] = vars(resp[key])
-                    except Exception as e:
-                        pass
-            except Exception as e:
-                print("error loading responsePayload " + obj.id + " " + str(e))
-            try:
-                headers = vars(obj.testData.responseHeaders)
-            except Exception as e:
-                print("error loading responseHeaders " + obj.id + " " + str(e))
-
-            data = {
-                "method": obj.testData.method,
-                "responsePayload": resp,
-                "statusCode": obj.testData.statusCode,
-                "responseHeaders": headers
-            }
-
-            url = obj.testData.url
-            if SampleData.objects.filter(url=url).exists():
-                SampleData.objects.filter(url=url).update(data=json.dumps(data))
+            if hasattr(obj.testData, 'method'):
```

Comment:
if hasattr(obj.testData, 'method'): ? why is this being decided based on method?

Reply:
the testData of single node test look like this:
![image](https://github.com/akto-api-security/vulnerable-apis/assets/37732050/4ca02420-79ac-4071-a182-aa1881808ef1)
So I used hasattr on method to check if it is a single node test or multiple node one.